modules/nf-core/purecn/intervalfile/main.nf

```diff
@@ -0,0 +1,43 @@
+def VERSION = '2.4.0' // PureCN outputs to stderr instead of stdout, and exits with 1 with --version
```

Comment:
Do these apply even if the command *has* a `--version` switch? PureCN has one, but it exits with status code 1, and that makes the whole process fails. No problem in doing that, but just asking.

Reply:
That's a weird way of defining the versions flag :). I don't know if there is any way you can circumvent this exit code for the versions flag (maybe using a bash function). Otherwise you can leave it as it is now indeed

Reply:
Can you adjust the manual version according to the guidelines which I stated in the first comment of this thread?